topics/ipfs/index.md

```diff
@@ -0,0 +1,19 @@
+---
+aliases: InterPlanetary-FileSystem, InterPlanetary FileSystem, Inter-Planetary FileSystem, IPFS, ipfs, InterPlanetary File System
+created_by: Juan Benet 
+display_name: IPFS
+logo: ipfs.png
+released: May 23, 2014
+short_description: InterPlanetary File System (IPFS) is a protocol and network designed to create a content-addressable, peer-to-peer method of storing and sharing hypermedia in a distributed file system
+topic: IPFS
```

Comment:
Can you make this lowercase? `topic: ipfs`

Reply:
Fixed